src/ui/button/expanded-button.tsx

```diff
@@ -0,0 +1,27 @@
+import clsx from 'clsx';
+import type { ButtonHTMLAttributes } from 'react';
+import { forwardRef } from 'react';
+
+import styles from './expanded-button.module.scss';
+
+type ExpandedButtonProps = {
+  isExpanded?: boolean;
+} & ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const ExpandedButton = forwardRef<HTMLButtonElement, ExpandedButtonProps>((
```

Comment:
Could you use this button also here
`src/components/data-tables/columns/ConjunctionEventHistoryColumns/ConjunctionEventHistoryColumns.tsx` ?